foreman-gutterball/rubygem-foreman_gutterball.spec

```diff
@@ -0,0 +1,94 @@
+%global gem_name foreman_gutterball
+
+# rubygems-devel defines a lot of nice macros, including:
+#   %{gem_dir}
+#   %{gem_instdir}
+#   %{gem_libdir}
+#   %{gem_cache}
+#   %{gem_spec}
+# that we'll be using below
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+Summary: Gutterball plugin for Foreman and Katello
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?dist}
+Group: Development/Languages
+License: GPLv3
+URL: http://katello.org
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+BuildArch: noarch
+
+%if "%{?scl}" == "ruby193"
+Requires: %{?scl_prefix}ruby-wrapper
+BuildRequires: %{?scl_prefix}ruby-wrapper
+%endif
+
+Requires: gutterball
+Requires: %{?scl_prefix}rubygem-katello
+Requires: %{?scl_prefix}rubygems
+BuildRequires: %{?scl_prefix}rubygems
+BuildRequires: %{?scl_prefix}rubygems-devel
+BuildRequires: foreman-plugin >= 1.8
+%if 0%{?rhel} > 6
+Requires: ruby(release) = 2.0.0
+BuildRequires: ruby(release) = 2.0.0
+%else
+%if "%{?scl}" == "ruby193"
+Requires: %{?scl_prefix}ruby(abi) = 1.9.1
+BuildRequires: %{?scl_prefix}ruby(abi) = 1.9.1
+%else
+Requires: ruby(abi) = 1.8
+BuildRequires: ruby(abi) = 1.8
+%endif
+%endif
```

Comment:
I think 35-46 can be simplified to https://github.com/Katello/katello-packaging/pull/5/files#diff-82ea4e79586571257d68905ed7b26176R20